pkg/api/usecase/wish_card/interactor.go

```diff
@@ -0,0 +1,201 @@
+package wish_card
```

Comment:
package名は、スネークケースやキャメルケースを利用しない形でお願い！
```suggestion
package wishcard
```

Reply:
承知です！パッケージなんやけど、自動生成のmockも改めて指定した方がいいですか？

Reply:
いや、mockに関してはそのままで大丈夫！！！
@akubi0w1